examples/into_file.rs

```diff
@@ -0,0 +1,17 @@
+use rodio::output_to_wav;
+use std::error::Error;
+use std::io::BufReader;
+
+/// Converts mp3 file to a wav file.
+/// This example does not use any audio devices
+/// and can be used in build configurations without `cpal` feature enabled.
+fn main() -> Result<(), Box<dyn Error>> {
+    let file = std::fs::File::open("assets/music.mp3")?;
+    let mut audio = rodio::Decoder::new(BufReader::new(file))?;
+
```

Comment:
we could show off some of rodio's audio pipeline here, maybe put the file through automatic gain control?